packages/material/src/complex/TableToolbar.tsx

```diff
@@ -79,14 +79,16 @@ export const TableToolbar = (
             </Grid>
             <Grid item>
               <Tooltip title='Delete'>
-                <Button
```

Comment:
why do we need to wrap this in a div?

Reply:
Because the tooltip doesn't pop up otherwise, see https://stackoverflow.com/a/18113961